### DISCUSSION TITLE
Эти дебилы реально отключили скробблинг?!

### DISCUSSION THREAD

**Discussion by kisoid** [Chimildic/YaMuTools#107] [Вопрос/ответ]
Не слушал музыку неделю. Сегодня заметил, что скробблинг на Ласт ФМ не работает.
Нашёл это:
https://pikabu.ru/story/yandeksmuzyika_ubrali_skrobbler_lastfm_12183962
https://yandexmusic.userecho.com/communities/45/topics/9833-vernut-skrobling-last-fm?page=89#comment-19155

Скажите, это не сон? Это реально произошло? Или я каких-то неправильных грибочков поел?

**Comment by Chimildic** [comment]
Бесполезно бороться за возвращение удаленных функций. 
Для ластфм на каждой платформе есть официальные и сторонние программы скробблинга. Например, [здесь](https://www.last.fm/ru/about/trackmymusic) или [здесь](https://4pda.to/forum/index.php?showtopic=887068).

**Reply by kisoid** [reply]
Не, придётся распрощаться с ЯМузыкой. 8 лет...
Возвращаюсь на foobar2000 + родной скробблер. Привет, нулевые :) Надоел этот гемор с облаками.
